contributors/guide/owners.md

```diff
@@ -115,6 +115,35 @@ emeritus_approvers:
 - emily    # 2019-01-05
 ```
 
+#### Cleanup
+
+In addition to Emeritus process above, from time to time it is necessary to
+prune folks from the OWNERS files who are not active in the community. This
+step is needed as the bots assign reviewers and approvers to pull requests and
+if these folks are not around anymore, there is a bad experience to folks who
+are trying to do the work. There is frustration all around as inactive folks
+are still inundated with github notifications / slack messages as they are
+listed in PR(s) and the folks doing the work will have to run around trying to
+find who may be still active in the area to help with progressing a PR.
+
+To find these inactive folks, we could look at a few places, one would be the
+number of contributions someone has made to the project as defined in devstats
+or even how many times someone has engaged in the PR review/approval process
+one way or another (commenting, running bot commands etc).
+
+Here's how we are counting someone as inactive
+- Devstats says someone has not had any contribution for a year as shown by
+  this
+  [dashboard](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All)
+- PR comments are less than 10 and Devstats count is less than 10
```

Comment:
```suggestion
- PR comments are less than 10 and Devstats count is less than 10 for a year
```

Or alternatively  say "year" on top for both items.

Reply:
```suggestion
```
(GitHub would not let me do a suggestion over this whole block)
The Devstats dashboard being referenced is just contributions, if we're using that as the source of truth I think this line can be removed.